python/GafferSceneUI/SetUI.py

```diff
@@ -176,22 +177,27 @@ def __setValue( plug, value, *unused ) :
 	with Gaffer.UndoScope( plug.ancestor( Gaffer.ScriptNode ) ) :
 		plug.setValue( value )
 
-def __spreadsheetTargetNode( plug ) :
-
-	# Find the first node the plug's column is connected to
-	# if it is a spreadsheet cell.
+def __targetPlug( plug ) :
```

Comment:
Now that we're doing this, I _think_ we should be able to remove the special case from https://github.com/GafferHQ/gaffer/blob/main/python/GafferUI/SpreadsheetUI/_Metadata.py#L259-L261. Could you test and remove if so?

Reply:
Yes, that looks good to remove. Done in 004c356fc5ae87b83e90795377a2f7b042e38c3d